data/migrations/20191118134921_cards.js

```diff
@@ -0,0 +1,41 @@
+exports.up = function(knex) {
+	return knex.schema.createTable("cards", tbl => {
+		tbl.increments();
+
+		tbl
+			.integer("user_id")
+			.unsigned()
+			.notNullable()
+			.references("id")
+			.inTable("users")
+			.onUpdate("CASCADE")
+			.onDelete("CASCADE");
+		tbl
+			.integer("event_id")
+			.unsigned()
+			.notNullable()
+			.references("id")
+			.inTable("events")
+			.onUpdate("CASCADE")
+			.onDelete("CASCADE");
```

Comment:
Shouldn't need an event_id since there's a many to many between cards and events.